### ISSUE TITLE
Failed with Json Output

### ISSUE THREAD

**Issue by secu77** [ShielderSec/webtech#10]
The "Output Json Format" (**--oj**) doesn't seem to work well (please, don't print a json data... write into a file).

```
root@kali:~/test# webtech -u http://127.0.0.1:8000 --oj demo.json
{'http://127.0.0.1:8000': {'tech': [{'name': 'SimpleHTTP', 'version': '0.6'}, {'name': 'Python', 'version': '2.7.16'}], 'headers': []}}
root@kali:~/test# webtech -u http://127.0.0.1:8000 --oj demo.json | jq
parse error: Invalid numeric literal at line 1, column 7
root@kali:~/test# ls demo.json
ls: no se puede acceder a 'demo.json': No existe el fichero o el directorio
```

**Comment by TheZ3ro** [comment]
I've pushed a new commit that fix the json syntax, It will be available in the new version of webtech soon
https://github.com/ShielderSec/webtech/commit/a9153958e28c4d6e5a797eb7f85378cd9a178d87

The `--oj` option doesn't support a file parameter right now, you need to use it this way:
```
$ webtech -u http://127.0.0.1:8000 --oj > demo.json  # save output to file
$ cat demo.json  # print file
$ webtech -u http://127.0.0.1:8000 --oj | jq    # print formatted output
$ cat demo.json | jq     # print formatted output from file
```

I will close this issue once the new version is out

**Comment by TheZ3ro** [comment]
Released a new 1.2.7 version on pip and on keybase
